networks/known_networks.go

```diff
@@ -859,6 +878,7 @@ var (
 
 	MappedNetworks = map[string]blockchain.EVMNetwork{
 		"SIMULATED":               SimulatedEVM,
+		"ANVIL":                   Anvil,
```

Comment:
@Tofel As I understand it, Anvil is a local/private blockchain node used for testing in the same way as "Simulated Geth." Therefore, you can use either one. Additionally, Anvil has the capability to fork existing networks. In this case, you start Anvil and point it to a remote EVM network using `--fork-url` and specify the block number.

Reply:
I do not agree. TBH I'd remove the word `Geth` from simulated and rename the whole thing as well. First of all it's no longer only `Geth`, second of all it's never again `simulated`. But that's a different discussion.

I also think that if you work a network you should use that `EVMNetwork` and not Anvil, so that chain id and other settings are correct (that might matter for CL node) and you'd only use Anvil endpoints.

For me with current approach we are making a logical error of treating a node (which just some EVM implementation) as if it was some distinct chain, which in my view, it is not.

Reply:
@Tofel 

1. I agree with the word "simulated". I suggest to use "Geth" and "Anvil" instead
2. Anvil and Geth networks are local testnet nodes as you mentioned. I agree it's not a real `EVMNetwork`. But then I don't think removing Geth and Anvil from `EVMNetworks` definitions (known_networks.go) is in the scope of this PR

Reply:
@skudasov pls help ^

Reply:
I'd just use one `EVMNetwork` for both, don't see any advantage is adding networks.

Reply:
@Tofel note that you may have different private keys or other fields

Reply:
We have many different names in known_networks.go now. Anvil is a simulator, but it's different from Geth. I think it's a good idea to differentiate them explicitly because strict client/network separation won't work in all cases. Anvil is a simulator but has network-specific capabilities, etc.